Automattic/woocommerce-payments
# Fix phrasing UPE missing currencies notice text

Fixes #2720

#### Changes proposed in this Pull Request



This PR fixes the phrasing of the missing currencies notice on the Add UPE methods modal. 

Previously, the notice contained:

- One method with one currency:

> Sofort require additional currencies, so we’ll add Euro (€) to your store. You can view & manage currencies later in settings.

- One method with two (or more) currencies:

> Przelewy24 (P24) require additional currencies, so we’ll add Euro (€) and Polish złoty (zł) to your store. You can view & manage currencies later in settings.

- Two (or more) methods with one currency:

> Sofort and giropay require additional currencies, so we’ll add Euro (€) to your store. You can view & manage currencies later in settings.

- Two (or more) methods with two (or more) currencies:

> Sofort and Przelewy24 (P24) require additional currencies, so we’ll add Euro (€) and Polish złoty (zł) to your store. You can view & manage currencies later in settings.


Now they contain:

> [Payment methods list] (require|requires) (an additional currency|additional currencies), so we’ll add [Currencies list] to your store. You can view & manage currencies later in settings.

And the list items are concatenated like this in the sentence:

> 1 : A
> 2 : A and B
> 3+: A, B, and C

#### Testing instructions

* Ensure you have UPE enabled in the dev tools page.
* Remove all currencies from Multi-currency page.
* Go to **Payments > Settings* page.
* Try to enable one or more UPE payment in the modal, observe the text when you select one or more providers from that list.

-------------------
